docs/src/bpmn-workflows/gateways.md

```diff
@@ -32,3 +33,26 @@ XML representation:
 ```
 
 BPMN Modeler: [Click Here](/bpmn-modeler/gateways.html#create-an-exclusive-gateway)
+
+## Parallel Gateway (AND)
+
+![workflow](/bpmn-workflows/parallel-gateway-example.png)
+
+A parallel gateway is only activated, when each of its incoming sequence flows has been signalled. Once activated, it signals all of its outgoing sequence flows.
```

Comment:
Can you describe more in detail how a parallel gateways works? How is the fork working? What happens to the payload?

Also, I think it's a bit confusing when you talk about that a sequence flow is signaled.